pkgs/applications/misc/scim/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, yacc, ncurses, libxml2 }:
+
+let
+  version = "0.1.8";
+in
+stdenv.mkDerivation rec {
+
+  name = "scim-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/andmarti1424/scim/archive/v${version}.tar.gz";
+    sha256 = "0qjixb1hzbdrypbmzwb2iaw5wp57kn7fmm1zpjp4gzjyanrhazs2";
+  };
+
+  buildInputs = [ yacc ncurses libxml2 ];
+
+  buildPhase = ''
+    cd src
+
+    sed "s,prefix=/usr,prefix=$out," Makefile
+    sed "s,-I/usr/include/libxml2,-I$libxml2," Makefile
+
+    make
+    export DESTDIR=$out
+    make install prefix=
```

Comment:
Sorry to say, this is nitpicking but `make install` should stay in `installPhase`. If you are lazy to do it no worries, can still merge this.